### ISSUE TITLE
Failed fetch with 1.9.0+

### ISSUE THREAD

**Issue by FilSalustri** [omnivore-app/obsidian-omnivore#179]
I've got an up-to-date obsidian installation, and I'm currently running 1.9.1 of the omnivore plugin.

I keep getting "Failed to fetch articles" both on startup, and if I do a fetch manually.
Nothing else in my installation has changed.
The error started appearing with 1.9.0.

Anyone else experiencing this?

**Comment by ChrisLAS** [comment]
+1 Came here to see if anyone was else was reporting this!

**Comment by FilSalustri** [comment]
Followup: I found that uninstalling the Omnivore plugin, removing the Omnivore folder from my vault, then reinstalling the plugin with a new API key seems to have overcome the problem.

There are, however, problems remaining, which I notice in the Obsidian Console:
1. there appear to be conflicts between `templater` and the Omnivore plugin; 
2. there are problems with article titles that include colons (and maybe other "special" characters); and
3. various other apparent issues that I do not understand cuz I'm not up on the programming skills needed to parse the console messages.

I could post my console output here. Would that help?

**Comment by MaryseBucking** [comment]
I'm experiencing this as well. Already uploaded a console log at #185. It started with version 1.9.1 and an update to 1.9.2 had no effect.

**Comment by StrollStars** [comment]
The problem still exists after updating to 1.9.3.

**Comment by FilSalustri** [comment]
So, @StrollStars I guess you're suggesting the workaround I stumbled across?
IE: delete my local Omnivore folder completely from my vault, then do a fresh fetch from the start?

**Comment by StrollStars** [comment]
@FilSalustri No, my synchronization rule is to sync the read articles. Last night, after syncing with the Omnivore client (not obsidian-omnivore), I found that the articles that were not synced became unread, marked as read, and all got synchronized without the need to make changes to the local Omnivore vault.

**Comment by FilSalustri** [comment]
I'm confused by "syncing with Omnivore client" vs "syncing with obsidian-omnivore". Could you point at a doc page or something that can clarify that distinction?

**Comment by StrollStars** [comment]
@FilSalustri We usually read through https://omnivore.app/ or the Omnivore app on our mobile phones. The progress is synchronized to the Omnivore official server. This is the syncing with Omnivore client.
When we use obsidian-omnivore, it is obsidian-omnivore that downloads data from the Omnivore server to the Obsidian client, in other words, syncing from the Omnivore server to the Obsidian client. This is syncing with obsidian-omnivore.
Simply put, our reading data is saved in the Omnivore client, synced to the Omnivore server from the client, and then synced from the Omnivore server to Obsidian (via obsidian-omnivore). That's roughly the process. From my perspective, it seems that the sync failure this time is due to an issue with the Omnivore server.

**Comment by FilSalustri** [comment]
Ah, that explains my confusion. I haven't installed the OM client. 
I just use the web version to read.
I use the OM extension for Chrome, and the app on Android, to **save** stuff.
I only **read** articles on my Macs, using the web app.

In any case, when I deleted the Omnivore folder in my Obsidian vault, reinstalled the Obsidian-omnivore plugin, then did a whole new fetch, that seemed to fix it.
Since then, the Obsidian plugin seems to be operating as expected.

**Comment by StrollStars** [comment]
@FilSalustri Congratulations, you have also resolved this issue.

**Comment by MaryseBucking** [comment]
@StrollStars Unfortunately for me this wasn't the solution. I already did this last week with no results. For me it is because of the content of the article that I included in the template. Also mentioned in issue #193. If I delete this part from my template, it syncs again.

**Comment by FilSalustri** [comment]
FWIW, I include the content too, but that's the only thing I've added to the template. I do _not_ have any frontmatter at all. 
Seems like my problem was completely unrelated to yours.

Actually one other thing I remember doing was resetting the template in the plugin to the default and then adding the `{{{ content }}}` back in. I remember thinking that the new default template was _very_ different from the old one - even though I didn't change it myself.

Anyways, I hope the problem gets resolved for you soon.

**Comment by StrollStars** [comment]
@FilSalustri Yes, it seems my issue is different from yours. Besides syncing and refreshing, I only marked as read. 'Marked as read' is my synchronization condition. Apart from that, no other operations were performed.
@MaryseBucking It seems your problem is more similar to @FilSalustri 's. I suggest you try @FilSalustri 's method. If it still doesn't work, you may need to wait for the developers to resolve it.

**Comment by sywhb** [comment]
hey guys, sorry for the delay in reply and i'm just back from holiday.

obsidian plugin `v1.9.4` is released to fix some syncing issue:

* add a config to the settings to disable sync on start
* update filter setting will update custom query accordingly and default to in:all

<img width="791" alt="Screenshot 2024-02-20 at 11 13 28 AM" src="https://github.com/omnivore-app/obsidian-omnivore/assets/12963965/847eb169-453f-4b51-8d7a-5ea1ea8b57fa">

<img width="791" alt="Screenshot 2024-02-20 at 11 13 37 AM" src="https://github.com/omnivore-app/obsidian-omnivore/assets/12963965/54c3f81d-2ae9-4bde-a3fb-8cdddeb9f998">

**Comment by sywhb** [comment]
another possible cause is invalid custom query, for example: label:\"some label\" needs to be changed to label:"some label" to make it work
